slackroll

```diff
@@ -1134,6 +1144,38 @@ def handle_dotnew_files_installation(dotnew_files): # Handles .new files present
     if len(dotnew_files) == 0:
         return
 
+    if slackroll_batch_mode:
+        handle_dotnew_files_installation_batch(dotnew_files)
+    else:
+        handle_dotnew_files_installation_interactive(dotnew_files)
+
+def handle_dotnew_files_installation_batch(dotnew_files): # Handles .new files present in packages in batch mode
+    # Iterate over every pair of .new files
+    for newfile in dotnew_files:
+        oldfile = old_file(newfile)
+        basenew = os.path.basename(newfile)
+        baseold = os.path.basename(oldfile)
+
+        try:
+            new_exists = os.path.exists(newfile)
+            old_exists = os.path.exists(oldfile)
+        except (OSError, IOError), err:
+            sys.stderr.write('ERROR: %s\n' % err)
+            break
+
+        if new_exists and old_exists: # Both
```

Comment:
I'd personally remove the blank lines in this if/else block but it's a matter of style.

Reply:
Yes, sorry, ruby style :)